### PR TITLE
feat: add frameworkless on openshift to the examples

### PR DESCRIPTION
**Pull request by aureamunoz**

### CODE REVIEW

**Comment by aureamunoz** [comment]
This can be reviewed @iocanel . Do you want me to add also the examples using `JIB`?

**Comment by aureamunoz** [comment]
Can you mark this PR as `Draft` @iocanel ? It seems I'm not able to do it

**Comment by cmoulliard** [comment]
This PR has been created long time ago Jul 2020. What is the status of this work ? 
Will it become part of the release 2.0 ?
@aureamunoz @iocanel

**Comment by iocanel** [comment]
> This PR has been created long time ago Jul 2020. What is the status of this work ?
> Will it become part of the release 2.0 ?
> @aureamunoz @iocanel

It's not mandatory for 2.0, but I'd like to get this in. We are working on it.

File: examples/rest-api-frameworkless-on-openshift-with-docker-example/readme.md
Comment (iocanel) [inline examples/rest-api-frameworkless-on-openshift-with-docker-example/readme.md:28]: name and pull policy should probably go.

File: examples/rest-api-frameworkless-on-openshift-with-docker-example/src/main/java/io.dekorate.fwlonopenshift/App.java
Comment (iocanel) [inline examples/rest-api-frameworkless-on-openshift-with-docker-example/src/main/java/io.dekorate.fwlonopenshift/App.java:14]: name and image pull policy should better go, as explained above.

**Review by iocanel — CHANGES_REQUESTED** [review]
I really really love the concept and the code looks great!

I've requrested some changes all styling related:
- remove commented out code.
- algin naming: Use `kubernetes` instead of `k8s`
- Use a smaller name in the examples: `frameworkles-` instead of `rest-api-frameworkless-`
- Use the common `io.dekorate.example` as the package for the examples
- Do not use collapsed packages in a signle directory.
- Remove configuration options that are environment specific (e.g. realted to minikube) or not absolutely needed (e.g. custom names).
- Use placeholders for dependency and plugin version in poms.

File: examples/frameworkless-on-openshift-with-docker-example/pom.xml
Comment (iocanel) [inline examples/frameworkless-on-openshift-with-docker-example/pom.xml:55]: We need to update this now that the package is changed.

File: examples/frameworkless-on-kubernetes-example/readme.md
Comment (iocanel) [inline examples/frameworkless-on-kubernetes-example/readme.md:24]: We need to align this with the current content of the class  (e.g. remove name, imagePullPolicy)

File: examples/frameworkless-on-openshift-with-docker-example/readme.md
Comment (iocanel) [inline examples/frameworkless-on-openshift-with-docker-example/readme.md:26]: Same as with the kubernetes example. We need to align this (remove uneeded options)

**Review by iocanel — CHANGES_REQUESTED** [review]
Added a couple of more remarks:
- align readme with actual code.
- fix a mainClass reference in the k8s example.

Other than that I think we are great!

**Comment by aureamunoz** [comment]
Finally the Openshift example uses the s2i build. Commits squashed, master rebased. It should be green. Would you like to take a look? Once the CI finish we can merge @iocanel.

**Review by iocanel — COMMENTED** [review]
LGTM, once CI is green we can merge.

**Pull request merged**
